test/DeleteCommandTest.py

```diff
@@ -124,14 +128,14 @@ def test_multi_del1(self):
         command = DeleteCommand(["1", "2"], self.todolist, self.out, self.error, _no_prompt)
         command.execute()
 
-        self.assertEqual(self.todolist.count(), 0)
+        self.assertEqual(self.todolist.count(), 2)
 
     def test_multi_del2(self):
         """ Test deletion of multiple items. """
         command = DeleteCommand(["1", "2"], self.todolist, self.out, self.error, _yes_prompt)
         command.execute()
 
-        self.assertEqual(self.todolist.count(), 0)
+        self.assertEqual(self.todolist.count(), 2)
```

Comment:
Same: checking just the count is not enough, also check the content of what's left.